pkg/core/model/message.go

```diff
@@ -14,65 +34,36 @@ const (
 	UpdateOperation        = "update"
 	ResponseOperation      = "response"
 	ResponseErrorOperation = "error"
-
-	ResourceTypePod        = "pod"
-	ResourceTypeConfigmap  = "configmap"
-	ResourceTypeSecret     = "secret"
```

Comment:
where do we plan to move these constants ??

Reply:
@sids-b  we can move to KubeEdge project. because In KubeEdge project, we can also see some definition about resourceType. such as 
```
	ResourceTypeService       = "service"
	ResourceTypeServiceList   = "servicelist"
	ResourceTypeEndpoints     = "endpoints"
	ResourceTypeEndpointsList = "endpointslist"
``` 
in https://github.com/kubeedge/kubeedge/blob/ae986072b147d22f35823d0894012d6e6577383e/common/constants/default.go#L50